### PR TITLE
add errors handling

### PR DESCRIPTION
**Pull request by mjudeikis**
Add some output before starting because if it fails before errors is not clear

### CODE REVIEW

File: src/go/cmd/azure-purge/main.go [hunk 1]
```diff
@@ -286,6 +307,7 @@ func purgeGroups() error {
 }
 
 func run() error {
+	fmt.Printf("Start azure-purge\n")
```
Comment (kargakis) [inline src/go/cmd/azure-purge/main.go]:
```go
fmt.Println("Start purging Azure resources")
```

**Comment by mjudeikis** [comment]
ptal @jim-minter

File: src/go/cmd/azure-purge/main.go [hunk 2]
```diff
@@ -71,6 +78,20 @@ func getClients() error {
 	return nil
 }
 
+func checkEnvConfig() error {
+	var errstrings []string
+	for _, env := range envList {
+		_, ok := os.LookupEnv(env)
+		if ok == false {
+			errstrings = append(errstrings, error.Error(fmt.Errorf("%s not set", env)))
```
Comment (jim-minter) [inline src/go/cmd/azure-purge/main.go]: `errstrings = append(errstrings, fmt.Sprintf("%s not set", env))`
or
`errstrings = append(errstrings, env + " not set")`

Comment (jim-minter) [reply src/go/cmd/azure-purge/main.go]: (`return fmt.Errorf("must set AZURE_* environment variables")` would have done as well, and would have been lighter ;)

Comment (mjudeikis) [reply src/go/cmd/azure-purge/main.go]: I'l take first one. Last do not print variable I missed but tells me to go rtfm :)  I like errors who tells me where I messed up :dagger:

File: src/go/cmd/azure-purge/main.go [hunk 3]
```diff
@@ -71,6 +78,20 @@ func getClients() error {
 	return nil
 }
 
+func checkEnvConfig() error {
+	var errstrings []string
+	for _, env := range envList {
+		_, ok := os.LookupEnv(env)
+		if ok == false {
```
Comment (jim-minter) [inline src/go/cmd/azure-purge/main.go]: `if _, ok := os.LookupEnv(env); !ok {` is the go idiom here fwiw

**Comment by jim-minter** [comment]
merging although this is surely fatter than it needs to be

**Pull request merged**